lib/opensrs/server.rb

```diff
@@ -11,6 +11,12 @@ class ConnectionError < OpenSRSError; end
   class TimeoutError < ConnectionError; end
 
   class Server
+    class << self
+      attr_accessor :ciphers
+    end
+    self.ciphers = ['AES128-SHA']
```

Comment:
Using a single cipher suite seems fragile, since the connection will fail if the server ever removes support for it.

I'd recommend starting with the "Broad Compatibility" list on the [OWASP cheat sheet](https://cheatsheetseries.owasp.org/cheatsheets/TLS_Cipher_String_Cheat_Sheet.html#openssl) and removing the `DHE-` suites from the list.

Reply:
```suggestion
    self.ciphers = 'TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA256'
```

Reply:
> fragile, since the connection will fail if the server ever removes support for it

Absolutely, hence why I wanted a list from sec people :)

Reply:
I tested that list, seem to work just fine. THanks!